src/chains/ethereum/options/src/wallet-options.ts

```diff
@@ -229,7 +229,7 @@ export const WalletOptions: Definitions<WalletConfig> = {
     conflicts: ["totalAccounts"]
   },
   totalAccounts: {
-    normalize,
+    normalize: rawInput => (rawInput > MAX_ACCOUNTS ? MAX_ACCOUNTS : rawInput),
```

Comment:
Rather than normalizing this to MAX_ACCOUNTS, should we validate and throw?

Reply:
It's already setup to handle `NaN` so I went for the least impact.

E.g.
``` bash
$ ganache -a roflcopter 
Available Accounts
==================
(no accounts unlocked)

```

Reply:
:+1: https://github.com/trufflesuite/ganache/pull/3361/files#diff-b465f04541f0c5e2ad842f50433df5af81c1bdd98b7921c558dbd30d635ca25dR234